core/src/main/java/com/amplifyframework/auth/AuthException.java

```diff
@@ -449,6 +502,24 @@ public FailedAttemptsLimitExceededException(Throwable cause) {
         }
     }
 
+    /**
+     *
+     */
+    public static class TooManyRequestsException extends AuthException {
+        private static final long serialVersionUID = 1L;
+        private static final String MESSAGE = "";
```

Comment:
The message should not be empty. 
```suggestion
        private static final String MESSAGE = "Failed since the user made too many requests.";
```